### PR TITLE
Update Data Connect regions

### PR DESCRIPTION
**Pull request by gladyscodes**
Adds new available Graph Data Connect regions.

### CODE REVIEW

File: concepts/data-connect-datasets.md [hunk 1]
```diff
@@ -99,13 +110,31 @@ Sinks are the output location that Azure Synapse or Azure Data Factory uses to p
...
 
 The following characteristics apply to sinks:
 
-- The output files are of format JSON lines. The output format is fixed, and there is no support for modifying the format of the output. However, you can use Azure Synapse or Azure Data Factory to copy the result of a Microsoft Graph Data Connect pipeline into another storage mechanism (such as Azure SQL Database).
 - Service Principal authentication is the only supported authentication mechanism for all sink types in a copy activity with Microsoft 365 as the source.
+
 - When using Azure Storage Blob as the sink, you must ensure that your application has Storage Blob Data Contributor access to the Azure Storage Blob location.
 
+- For Copy Activity, the output files are of format JSON lines. The output format is fixed, and there is no support for modifying the format of the output. However, you can use Azure Synapse or Azure Data Factory to copy the result of a Microsoft Graph Data Connect pipeline into another storage mechanism (such as Azure SQL Database).
```
Comment (Lauragra) [inline concepts/data-connect-datasets.md:121]: Is this just a general reference to the activity of copying the output, or is it a reference to something that's a proper noun? I'm not sure why Copy Activity is capitalized. Can we say that the output lines are formatted as JSON, in the second sentence? Suggested revision:

... the output files are formatted as JSON. This format is fixed and modifying the format is not supported.

File: concepts/data-connect-datasets.md [hunk 2]
```diff
@@ -99,13 +110,31 @@ Sinks are the output location that Azure Synapse or Azure Data Factory uses to p
...
+
+- MDF: [Microsoft Graph Data Connect overview - Microsoft Graph | Microsoft Learn](/graph/data-connect-concept-overview)
```
Comment (Lauragra) [inline concepts/data-connect-datasets.md:123]: What is MDF and why do we have a link here to the Data Connect overview topic?

File: concepts/data-connect-datasets.md [hunk 3]
```diff
@@ -99,13 +110,31 @@ Sinks are the output location that Azure Synapse or Azure Data Factory uses to p
...
+
+    - Output can be in parquet format. Find more information on the supported data transformations [here](/azure/data-factory/data-flow-flatten).
```
Comment (Lauragra) [inline concepts/data-connect-datasets.md:125]: Revise the second sentence as:
For details about the supported data transformations, see [Flatten transformation in mapping data flow](/azure/data-factory/data-flow-flatten).

File: concepts/data-connect-datasets.md [hunk 4]
```diff
@@ -99,13 +110,31 @@ Sinks are the output location that Azure Synapse or Azure Data Factory uses to p
...
+
 - When using Azure Storage Blob as the sink, you must ensure that your application has Storage Blob Data Contributor access to the Azure Storage Blob location.
 
+- For Copy Activity, the output files are of format JSON lines. The output format is fixed, and there is no support for modifying the format of the output. However, you can use Azure Synapse or Azure Data Factory to copy the result of a Microsoft Graph Data Connect pipeline into another storage mechanism (such as Azure SQL Database).
+
+- MDF: [Microsoft Graph Data Connect overview - Microsoft Graph | Microsoft Learn](/graph/data-connect-concept-overview)
+
+    - Output can be in parquet format. Find more information on the supported data transformations [here](/azure/data-factory/data-flow-flatten). 
+
+    - MGDC on MDF supports direct output of the data into Azure SQL DB.
```
Comment (Lauragra) [inline concepts/data-connect-datasets.md:127]: Per branding guidelines, we don't use MGDC, we spell out as Microsoft Graph Data Connect or we can shorten to Data Connect after first mention/when the context is clear.

File: concepts/data-connect-datasets.md [hunk 5]
```diff
@@ -99,13 +110,31 @@ Sinks are the output location that Azure Synapse or Azure Data Factory uses to p
...
+
+&nbsp;
```
Comment (Lauragra) [inline concepts/data-connect-datasets.md:129]: Please add a sentence here introducing the table.

File: concepts/data-connect-troubleshooting.md [hunk 6]
```diff
@@ -70,40 +70,47 @@ You can resolve this issue with an SSP request: `INTERNT PROXY (SWG) - EXCEPTION
 
 ## Issues adding network IP address to allow list with Azure integration runtime
 
-If the destination storage account needs to be closed for public access, you need to allow access for a particular set of Azure service IP addresses. Customers will need to allow list their IPs based on their region, the region of tenancy they want to extract data from, and their Azure IR region. To do this:
+If the destination storage account needs to be closed for public access, you need to allow access for a particular set of Azure service IP addresses. Customers will need to allow list IPs based on the targeted office region. To do this:
```
Comment (Lauragra) [inline concepts/data-connect-troubleshooting.md:73]: Is this a reference to Office the product? If so, we should capitalize it, and should it actually be Microsoft 365?

File: concepts/data-connect-troubleshooting.md [hunk 7]
```diff
@@ -70,40 +70,47 @@ You can resolve this issue with an SSP request: `INTERNT PROXY (SWG) - EXCEPTION
...
 
-1. Find an Office to Azure region mapping. To look up which Office region you will be extracting user data from, see [Regions](/graph/data-connect-datasets#regions).
+1. Find an Office-to-Azure region mapping. To look up which Office region you will be extracting user data from, see the table below.
```
Comment (Lauragra) [inline concepts/data-connect-troubleshooting.md:75]: Global: We don't use "below" to indicate where something exists in a topic. You can revise as "the following table".

File: concepts/data-connect-troubleshooting.md [hunk 8]
```diff
@@ -70,40 +70,47 @@ You can resolve this issue with an SSP request: `INTERNT PROXY (SWG) - EXCEPTION
...
 
-2. After you find the Office to Azure mapping, you need to determine the correct and compatible location of your destination storage account. You can look up how to configure your Azure storage account and [grant access from an internet IP range](/azure/storage/common/storage-network-security?tabs=azure-portal#grant-access-from-an-internet-ip-range).
+2. After you find the Office to Azure mapping, you need to determine the compatible location of your destination storage account (see table below). You can look up how to configure your Azure storage account and [grant access from an internet IP range](/azure/storage/common/storage-network-security?tabs=azure-portal#grant-access-from-an-internet-ip-range). 
 
-   a. Use the following table to select an Azure storage account that meets the criteria.
+> [!*NOTE]
+> This indicates the Azure regions that may NOT be used per region for the destination storage **when it is closed for public access**. This is also the region for which the IP addresses need to be allowlisted to allow data delivery. To find IP ranges, see [Azure IP Ranges and Service Tags](https://www.microsoft.com/download/details.aspx?id=56519).
```
Comment (Lauragra) [inline concepts/data-connect-troubleshooting.md:83]: Please change "allowlisted" to "added to the allow list".

File: concepts/data-connect-troubleshooting.md [hunk 9]
```diff
@@ -70,40 +70,47 @@ You can resolve this issue with an SSP request: `INTERNT PROXY (SWG) - EXCEPTION
...
 
-    > [!NOTE]
-    > - At this point, customers can understand and configure the region they want to extract users from (what their Office to Azure region mapping is).
-    > - Customers can understand which region their destination storage account can't be in.
-    > - Based on a compatible destination storage account, customers can usethe  information to understand which IP addresses they need to add to the allow list. 
+&nbsp;
+
+| Office region | Azure region | Alternate Azure regions to use |
+| ------------- | ------------ | ------------------------------ |
+| **Asia-Pacific**   | <ul><li>East Asia</li><li>Southeast Asia*</li></ul> |  |
```
Comment (Lauragra) [inline concepts/data-connect-troubleshooting.md:95]: The third column is empty for a number of entries. Please don't leave them empty; either add the content or put NA.

File: concepts/data-connect-troubleshooting.md [hunk 10]
```diff
@@ -129,6 +136,20 @@ The following example describes how to troubleshoot network access issue:
 
 8.	 The user lists these IP addresses and moves the destination storage to North Europe because the Office region is EUR, and the Azure region is West Europe. 
 
+## Issues with running your pipeline using mapping data flows 
+
+First time runs of [MGDC-MDF](/azure/data-factory/connector-office-365?tabs=data-factory#transform-data-with-the-microsoft-365-connector) activity for a new dataset are expected to fail with the “Consent Pending” error. This triggers a consent request for the tenant admin who can use [Privileged Access Management](/graph/data-connect-quickstart?tabs=Microsoft365&tutorial-step=6) to review and approve/decline the data access request.
```
Comment (Lauragra) [inline concepts/data-connect-troubleshooting.md:141]: The link text MGDC-MDF doesn't match the link target title, which is Transform data with the Microsoft 365 connector. Can you please clarify that here? For SEO and accessibility, link text should match the title of the target you're linking to.

File: concepts/data-connect-troubleshooting.md [hunk 11]
```diff
@@ -129,6 +136,20 @@ The following example describes how to troubleshoot network access issue:
 
 8.	 The user lists these IP addresses and moves the destination storage to North Europe because the Office region is EUR, and the Azure region is West Europe. 
 
+## Issues with running your pipeline using mapping data flows 
+
+First time runs of [MGDC-MDF](/azure/data-factory/connector-office-365?tabs=data-factory#transform-data-with-the-microsoft-365-connector) activity for a new dataset are expected to fail with the “Consent Pending” error. This triggers a consent request for the tenant admin who can use [Privileged Access Management](/graph/data-connect-quickstart?tabs=Microsoft365&tutorial-step=6) to review and approve/decline the data access request.  
+
+1. The consent request is only valid for 24-hrs. Please contact your tenant admin to approve within this timeframe.
```
Comment (Lauragra) [inline concepts/data-connect-troubleshooting.md:143]: Please change "24-hrs" to "24 hours".

File: concepts/data-connect-troubleshooting.md [hunk 12]
```diff
@@ -129,6 +136,20 @@ The following example describes how to troubleshoot network access issue:
...
 
+## Issues with running your pipeline using mapping data flows 
+
+First time runs of [MGDC-MDF](/azure/data-factory/connector-office-365?tabs=data-factory#transform-data-with-the-microsoft-365-connector) activity for a new dataset are expected to fail with the “Consent Pending” error. This triggers a consent request for the tenant admin who can use [Privileged Access Management](/graph/data-connect-quickstart?tabs=Microsoft365&tutorial-step=6) to review and approve/decline the data access request.  
+
+1. The consent request is only valid for 24-hrs. Please contact your tenant admin to approve within this timeframe.  
+
+    a. If not approved in that timeframe, subsequent runs will fail with the same error and regenerate a consent request.
+
+    b. Once approved, the pipeline can be rerun at any time to retrieve data.
```
Comment (Lauragra) [inline concepts/data-connect-troubleshooting.md:147]: Change "Once" here to "When".

File: concepts/data-connect-troubleshooting.md [hunk 13]
```diff
@@ -129,6 +136,20 @@ The following example describes how to troubleshoot network access issue:
...
+
+    ![An image that shows the error the first time MGDC-MDF runs](images/data-connect-mdf-error.png)
+
+2. Please verify that the destination storage is correctly setup to allow the appId to write data into it.
```
Comment (Lauragra) [inline concepts/data-connect-troubleshooting.md:151]: Revise this sentence as:
Verify that the destination storage is set up correctly to allow the app to write data into it.

I think you mean the "app" not the "app ID", right?

Comment (gladyscodes) [reply concepts/data-connect-troubleshooting.md]: Hi @Lauragra thank you for the feedback I just addressed all your comments, thank you! 😊

**Review by Lauragra — APPROVED** [review]

**Pull request merged**